src/WooCommerce.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\Ssofact\WooCommerce.
+ */
+
+namespace Netzstrategen\Ssofact;
+
+class WooCommerce {
+
+  /**
+   * @implements woocommerce_before_customer_login_form
+   */
+  public static function woocommerce_before_customer_login_form() {
+    ob_start();
+  }
+
+  /**
+   * @implements woocommerce_after_customer_login_form
+   */
+  public static function woocommerce_after_customer_login_form() {
+    $output = ob_get_clean();
+    $authorize_uri = static::getAuthorizeUrl();
+    $server_domain = 'stage-login.stimme.de';
```

Comment:
too me looks like a class constant

Reply:
I just moved this code to the new `WooCommerce` class, it was written by Daniel. So I'll leave it as it is.